sktime/model_selection.py

```diff
@@ -225,3 +233,35 @@ def __init__(self, estimator, param_grid, scoring=None, fit_params=None,
             # using mean squared error as default for regressors
             elif isinstance(self.estimator, BaseRegressor):
                 self.scoring = make_scorer(mean_squared_error)
+
+class SKTimeCV(ABC):
```

Comment:
Do we still need this class?

Reply:
SKTimeCV? Yes, I believe so, PredefinedSplit and other custom CV objects should inherit from it.

Reply:
Instead of creating new base classes for cv iterators, I'd rather integrate our objects into the existing sklearn hierarchy of CV classes. (Also currently the inheritance does not do anything unless I'm not seeing something - if the split method should be abstract, it needs the @abstractmethod decorator, in either case, we initially agreed to avoid abstract methods for now if I remember correctly)

Reply:
I forgot the decorator in the the abstract class, this is a mistake.

I am thinking along the lines of the extension guidelines, i.e. if a user wants to write a new CV object they would inherit from SKTimeCV and implement spilt(). At this time, the code is fairly simple so might be an overkill but this can be a future-proof solution.

How do you propose to integrate the cv classes in the sklearn hierachy, by inheriting from BaseCrossValidator? I don't think that this will work particularly well in our case as the split() method that is inheritted will not be appropriate for our predefined splits use case.

In any case, let's delete the abstract class, it doesn't do anything at the moment and there is no scope to write more CV methods in the near future.